src/sentry/static/sentry/app/views/settings/organizationRelays/dialogs/form.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {t} from 'app/locale';
+import TextField from 'app/components/forms/textField';
+import TextareaField from 'app/components/forms/textareaField';
+import space from 'app/styles/space';
+
+import FormField from './formField';
+
+type FormField = 'name' | 'description' | 'publicKey';
+type Values = Record<FormField, string>;
+
+type Props = {
+  values: Values;
+  errors: Partial<Values>;
+  disables: Partial<Record<FormField, boolean>>;
+  onValidate: (field: FormField) => () => void;
+  onChange: (field: FormField, value: string) => void;
+};
+
+const Form = ({values, errors, onValidate, onChange, disables}: Props) => (
+  <Wrapper>
+    <FormField label={t('Name')} isFullWidth>
+      <StyledTextField
+        name="name"
+        onChange={(value: string) => {
+          onChange('name', value);
+        }}
+        value={values.name}
+        onBlur={onValidate('name')}
+        error={errors.name}
+        disabled={disables.name}
+      />
+    </FormField>
+    <FormField label={t('Key')} isFullWidth>
```

Comment:
The `FormField` in `app/views/settings/components/form` has a prop for these kind of layout: https://storybook.getsentry.net/branches/master/index.html?path=/story/forms-form--default